README.md

```diff
@@ -124,6 +127,49 @@ to production private keys.
 * setInstance (instance) - Set a config instance.
 
 
+
+## Rotating keys
+
+In the case you have keys that must be rotated, you can use the convert with `backup` option. The process would
+require you to:
+
+1. Load config with old private key.
+2. Convert with new private key, setting `backup` to `true`.
+3. Deploy your config change.
+4. Rotate your private keys.
+5. Load config with new private key.
+6. Run `dropBackup`.
+7. Deploy your final config change.
+
+In CLI, would look something like:
+
+```
+  config-shield
+  enter path of config> secure-config.json
+  enter path of private key> old.key
+  > convert
+  enter path of private key> new.key
+  backup (press enter to disable backup)> true

```

Comment:
The actual full message is more detailed, I just shorthanded the readme. I can update